### DISCUSSION TITLE
Example project can't run successfully

### DISCUSSION THREAD

**Discussion by Chralay** [0xTheProDev/Operational-Transformation#113] [Bug Report]
Hey,
I try to use this project, but I get an error when I run it with `yarn start` in example project：
(node:51187) MetadataLookupWarning: received unexpected error = Response timeout while trying to fetch http://169.254.169.254/computeMetadata/v1/instance (over 3000ms) code = UNKNOWN

**firebase-debug.json** show :
[debug] [2022-06-20T08:12:51.908Z] <<< HTTP RESPONSE 400 {"cache-control":"no-cache, no-store, max-age=0, must-revalidate","date":"Mon, 20 Jun 2022 08:12:51 GMT","pragma":"no-cache","expires":"Mon, 01 Jan 1990 00:00:00 GMT","content-type":"application/json; charset=utf-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","server":"scaffolding on HTTPServer2","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3=\":443\"; ma=2592000,h3-29=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}

[debug] [2022-06-20T08:12:52.323Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects/undefined {"error":{"code":401,"message":"Request is missing required authentication credential. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.","status":"UNAUTHENTICATED","details":[{"@type":"[type.googleapis.com/google.rpc.ErrorInfo](http://type.googleapis.com/google.rpc.ErrorInfo)","reason":"CREDENTIALS_MISSING","domain":"[googleapis.com](http://googleapis.com/)","metadata":{"method":"google.firebase.service.v1beta1.FirebaseProjectService.GetFirebaseProject","service":"[firebase.googleapis.com](http://firebase.googleapis.com/)"}}]}}

**Comment by 0xTheProDev** [comment]
Hi @Clara-hy,

From the logs you have pasted, it appears to be an access issue. Just to be sure, have you done this step from `yarn start`:
https://github.com/Progyan1997/Operational-Transformation/tree/main/examples/collaborative-editors#prerequisites

If yes, can you ensure you have logged in or using the right database. As I see an `undefined` in the URL, which should not be the case.
